packages/backend/src/tests/outgoingPayment.ts

```diff
@@ -15,7 +15,15 @@ export async function createOutgoingPayment(
     'quoteId'
   >
```

Comment:
If you change `options` to :point_down: you won't need the changes below
```
  options: CreateTestQuoteOptions &
    Omit<CreateOutgoingPaymentOptions, 'quoteId'>
```
but then it looks like there's some calls to `createOutgoingPayment` that would need to updated.

Reply:
🤔 I _think_ we should keep the types as it was before, since our changes are affecting only the `createQuote` function ?

Reply:
What do you think if we refactor `createOutgoingPayment` later on, to something like:
```ts
function createOutgoingPayment(
  deps: IocContract<AppServices>,
  quoteOptions: CreateTestQuoteOptions,
  outgoingPaymentOptions: Omit<CreateOutgoingPaymentOptions, 'quoteId'>
): Promise<OutgoingPayment>
```